### PR TITLE
[desktop] Add 1337x torrent icon

### PR DESCRIPTION
**Pull request by Aholicknight**
This icon has been cropped to remove the 1337 part and kept the X part of the logo. The icon was grayscaled to fit the colour scheme of the other icons and cropped to 20x20.

### CODE REVIEW

**Comment by gubatron** [comment]
Thank you also for this submission, in order to merge:

1. We need it in 16x16.
2. We need the source icons to be in this RGB color: `#8ea2b4`
3. Please rename the file to be `1337_source.png`
4. If you can add the code in the [source renderer](https://github.com/frostwire/frostwire/blob/master/desktop/src/com/limegroup/gnutella/gui/search/SourceRenderer.java#L49) it'd be awesome so that your pull request is 100% complete, the lines there should give you an idea.

**Comment by Aholicknight** [comment]
Fixing it right now.

File: desktop/src/com/limegroup/gnutella/gui/search/SourceRenderer.java [hunk 1]
```diff
@@ -48,6 +48,7 @@
             sourceIcons.put("eztv", GUIMediator.getThemeImage("eztv_off"));
             sourceIcons.put("zooqle", GUIMediator.getThemeImage("zooqle_source"));
             sourceIcons.put("default", GUIMediator.getThemeImage("seeding_small_source"));
+            sourceIcons.put("1337x", GUIMediator.getThemeImage("1337_source")); // Thank you AholicKnight for the icon.
```
Comment (gubatron) [inline desktop/src/com/limegroup/gnutella/gui/search/SourceRenderer.java:51]: nice! Thank you@AholicKnight

**Comment by gubatron** [comment]
Merged!

**Comment by gubatron** [comment]
Screen Shot 2020-11-16 at 10 21 52 PM
it worked :) Thanks again.

**Comment by Aholicknight** [comment]
You are welcome.

**Pull request merged**